<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialWidget</class>
 <widget class="QWidget" name="MaterialWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox
{
    border: 1px solid gray;
    border-bottom-left-radius: 5px;
    border-bottom-right-radius: 5px;
    margin-top: 2ex;
}

QGroupBox::title
{
    background: #bbb;
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 2px 800px 2px 3px;
}

QSlider
{
    height: 18px;
}

QSlider::groove:horizontal
{
    border: 1px solid #999999;
    height: 2px;
}

QSlider::handle:horizontal
{
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
    border: 1px solid #5c5c5c;
    width: 10px;
    margin: -5px 0;
    border-radius: 5px;
}

QRadioButton
{
    color: black;
    height: 14px;
}

QRadioButton::indicator
{
    color: black;
    width: 10px;
    height: 10px;
    top: 1;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="DropLabel" name="Material">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string>material</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Name">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="Reset">
         <property name="text">
          <string>@</string>
         </property>
         <property name="icon">
          <iconset resource="../material/materialplugin.qrc">
           <normaloff>:/materialplugin/reset.png</normaloff>:/materialplugin/reset.png</iconset>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QGroupBox" name="TintGroup">
         <property name="styleSheet">
          <string notr="true">QGroupBox::title
{
    background: none;
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 3px 0 3px;
    left: 8px;
}</string>
         </property>
         <property name="title">
          <string>Tint</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QcDoubleSlider" name="TintRedSlider">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QcDoubleSpinBox" name="TintRedSpinner">
              <property name="minimumSize">
               <size>
                <width>52</width>
                <height>0</height>
               </size>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="TintRedLabel">
              <property name="minimumSize">
               <size>
                <width>8</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>R</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QcDoubleSlider" name="TintGreenSlider">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QcDoubleSpinBox" name="TintGreenSpinner">
              <property name="minimumSize">
               <size>
                <width>52</width>
                <height>0</height>
               </size>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="TintGreenLabel">
              <property name="minimumSize">
               <size>
                <width>8</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>G</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QcDoubleSlider" name="TintBlueSlider">
              <property name="maximum">
               <number>100</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QcDoubleSpinBox" name="TintBlueSpinner">
              <property name="minimumSize">
               <size>
                <width>52</width>
                <height>0</height>
               </size>
              </property>
              <property name="singleStep">
               <double>0.100000000000000</double>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="TintBlueLabel">
              <property name="minimumSize">
               <size>
                <width>8</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>B</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>51</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QcDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qcdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>DropLabel</class>
   <extends>QLabel</extends>
   <header>droplabel.h</header>
  </customwidget>
  <customwidget>
   <class>QcDoubleSlider</class>
   <extends>QSlider</extends>
   <header>qcdoubleslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../material/materialplugin.qrc"/>
 </resources>
 <connections/>
</ui>
